runtime/astar/src/weights/frame_system_extensions.rs

```diff
@@ -0,0 +1,122 @@
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Astar. If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for `frame_system_extensions`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 46.1.0
+//! DATE: 2025-05-04, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `gh-runner-01-ovh`, CPU: `Intel(R) Xeon(R) E-2236 CPU @ 3.40GHz`
+//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("astar-dev")`, DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/astar-collator
+// benchmark
+// pallet
+// --chain=astar-dev
+// --steps=50
+// --repeat=20
+// --pallet=frame_system_extensions
+// --extrinsic=*
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --output=./benchmark-results/astar-dev/system_extensions_weights.rs
+// --template=./scripts/templates/weight-template.hbs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+#![allow(dead_code)]
+
+use frame_support::{traits::Get, weights::Weight};
+use core::marker::PhantomData;
+
+/// Weight functions for `frame_system_extensions`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> frame_system::ExtensionsWeightInfo for WeightInfo<T> {
+	fn check_genesis() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `30`
+		//  Estimated: `0`
+		// Minimum execution time: 3_436_000 picoseconds.
+		Weight::from_parts(3_544_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+	}
+	fn check_mortality_mortal_transaction() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `68`
+		//  Estimated: `0`
+		// Minimum execution time: 6_645_000 picoseconds.
+		Weight::from_parts(6_812_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+	}
+	fn check_mortality_immortal_transaction() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `68`
+		//  Estimated: `0`
+		// Minimum execution time: 6_742_000 picoseconds.
+		Weight::from_parts(6_930_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+	}
+	fn check_non_zero_sender() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 455_000 picoseconds.
+		Weight::from_parts(490_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+	}
+	/// Storage: `System::Account` (r:1 w:1)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	fn check_nonce() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `102`
+		//  Estimated: `3593`
+		// Minimum execution time: 6_711_000 picoseconds.
+		Weight::from_parts(7_001_000, 0)
+			.saturating_add(Weight::from_parts(0, 3593))
+			.saturating_add(T::DbWeight::get().reads(1))
+			.saturating_add(T::DbWeight::get().writes(1))
```

Comment:
Do you know perhaps where the `System::Account` change weight was accounted for before?

Reply:
From my understanding, it was accounted as a constant for all signed extensions in extrinsic base weight. As former signed extensions were not allowed to have complex operations unlike new transaction extensions

Reply:
I thought as well but checked the recent history of the files with base extrinsic weights - there's no PoV, and total weight of it is less than ~1 write according to RocksDB. 🤷 

I was asking just to make sure we don't count it twice, accidentally.

Reply:
Yeah that means we were charging less all this time